src/deluge/gui/views/view.cpp

```diff
@@ -203,6 +204,22 @@ ActionResult View::buttonAction(deluge::hid::Button b, bool on, bool inCardRouti
 		}
 	}
 
+	// Load button for MIDI learn
+	else if (b == LOAD && currentUIMode == UI_MODE_MIDI_LEARN) {
```

Comment:
this should probably be done in the global midi commands menu instead

Reply:
It's done there already. But you have this as an alternative, that's it, holding Load +learn and sending some midi message. It can be removed from this place if you think it shouldn't be here and leave it just in the main menu for midi commands.

Reply:
Oh ok, must have overlooked it